apps/src/applab/AppLabView.jsx

```diff
@@ -91,7 +89,8 @@ var AppLabView = React.createClass({
 
     var topPaneHeight = this.props.instructionsHeight;
     var totalHeight = topPaneHeight + this.refs.codeWorkspace.getContentHeight();
-    var instructionsContentHeight = this.refs.topInstructions.getContentHeight();
+    var topInstructions = this.refs.topInstructions.getWrappedInstance();
+    var instructionsContentHeight = topInstructions.getContentHeight();
```

Comment:
Interesting ramification of connecting a component - can no longer directly access members without passing in `withRef: true` to connect, and then calling `getWrappedInstance`.